src/styles/dialog.css

```diff
@@ -3,8 +3,8 @@
  * @author Jamie jamie.h@inwinstack.com
  */
 
-.md-confirm-dialog-warn-theme md-dialog-actions button:last-child {
-  color: #FF6D00;
+.md-default-theme md-dialog-actions button:last-child {
```

Comment:
You should use `.md-confirm-dialog-warn-theme` here.
[As before discuss](https://github.com/inwinstack/s3-portal-ui/pull/21), we need add `.theme('confirm-dialog-warn')` at javascript for highlight the button, please refer the confirm dialog of [codepen](http://codepen.io/jamie3312/pen/dMzogN?editors=1010). Thanks.